decidim-budgets/app/packs/src/decidim/budgets/exit_handler.js

```diff
@@ -82,7 +84,7 @@ $(() => {
   // logout form submit and so that it changes the exit link text. This does
   // not trigger the document link click listener because it has the
   // data-method attribute to trigger a form submit event.
-  $(".header a.sign-out-link").on("click", (event) => {
+  $("[href='/users/sign_out']").on("click", (event) => {
```

Comment:
Could we add the url to the decidim config variable ?

Reply:
Right!